misc/update/nix/tmux/monitor.php

```diff
@@ -217,7 +217,7 @@
 						$stamp = 'UNIX_TIMESTAMP(MIN(dateadded))';
 
 						switch (true) {
-							case strpos($tbl, 'collections_') !== false:
+							case strpos($tbl, 'collections') !== false:
```

Comment:
Is this intended to match "collections_*" and "multigroup_collections"? It's possible this would come back and bite us if we ever added another table with 'collections' in the name (same applies to the following 'binaries' and 'parts' checks too). It would be safer to use a match at pos 0 and be more specific with the names. i.e.
case strpos($tbl, 'collections') === 0 || strpos($tbl, 'multigroup_collections') === 0:
assuming the intent is to match both. This way the name has to start at column 0 and will avoid any we add later that don't